Template10 (Validation)/Validation/ValidatableViewModelBase.cs

```diff
@@ -9,16 +10,17 @@
 
 namespace Template10.Validation
 {
-    public abstract class ModelBase : BindableBase, IModel
+    public abstract class ValidatableViewModelBase : ViewModelBase, IModel
     {
-        public ModelBase()
+        public ValidatableViewModelBase()
```

Comment:
I think he meant to rename the`ValidatableViewModelBase` to `ValidatableModelBase`